includes/Data/Brands.php

```diff
@@ -87,6 +90,7 @@ public static function get_config( Container $container ) {
 						'woocommerce_default_country' => 'US:AZ',
 						'woocommerce_currency' => 'USD',
 					),
+					'wondercartBuyNow' => 'https://my.bluehost.com/hosting/app?utm_source=wp-marketplace&utm_medium=brand-plugin&utm_campaign=wordpress-ad&utm_content=buynow#/marketplace/product'
```

Comment:
Everything in the Brands.php file should be configured in the brand plugins instead. This ensures that key things like these brand-specific URLs are located in a single location rather than scattered across multiple modules. We'll leave this as is for now and create a request to address this later.